hudi-common/src/main/java/org/apache/hudi/common/util/FSUtils.java

```diff
@@ -532,6 +532,13 @@ public static Path getPartitionPath(Path basePath, String partitionPath) {
     return ((partitionPath == null) || (partitionPath.isEmpty())) ? basePath : new Path(basePath, partitionPath);
   }
 
+  /**
+   * Get HDFS full partition path (e.g. hdfs://ip-address:8020:/<absolute path>)
+   */
+  public static String getHDFSFullPartitionPath(FileSystem fs, Path partitionPath) {
```

Comment:
nit : rename to `getDFSFullPartitionPath()` since this is agnostic of hdfs itself.

Reply:
Done.